frappe/model/delete_doc.py

```diff
@@ -162,9 +162,6 @@ def delete_doc(
 				except ImportError:
 					pass
 
-			# delete user_permissions
-			frappe.defaults.clear_default(parenttype="User Permission", key=doctype, value=name)
-
```

Comment:
The reason why a translation test is failing is because this is no longer being called of deletion of the Translation document. This code internally reaches this line and triggers `frappe.clear_cache` for everything, including `frappe.local.cache` which is the culprit in the failing test.

Reply:
Fixed in https://github.com/frappe/frappe/pull/17304

Reply:
https://github.com/frappe/frappe/pull/17353  this should fix state issues in the tests.